### ISSUE TITLE
Multi-Source Scan host list

### ISSUE THREAD

**Issue by bclarhk** [quipucords/quipucords#1030]
## Specify type:
- Bug

### Priority:
- High
___

## Description:
Scanning multiple networks returns a combined list of hosts for an individual network.

___

## Bug Report
Create 2 network sources
- Net#1: 10.10.181.[98:178]
- Net#2:  10.10.181.[0:98]

Select them both and click "Scan"

The interface should return the correct number of hosts (6) for each source, but the list of hosts is combined. The 10.10.181.98 address is also listed twice.

Going to the "Scans" page and expanding the list of hosts also fails. See second attachment.

### Version Information:
#1028 

![image](https://user-images.githubusercontent.com/32572198/37412669-c246f434-277b-11e8-92a4-6184f867bc85.png)

![image](https://user-images.githubusercontent.com/32572198/37421212-1c109b24-278f-11e8-958c-1f520cb50e32.png)

**Comment by bclarhk** [comment]
The "Details" download also has the same address twice. The "Summary" download has the correct 11 addresses.

**Comment by cdcabrera** [comment]
@bclarhk after #1041 merged appears to have corrected some of the totals. The issue appears resolved for sources, can you confirm?

New issue appears to have popped up on Scans view, only a single IP address appears to be displaying for inspection scans:

![screen shot 2018-03-15 at 4 24 53 pm](https://user-images.githubusercontent.com/3761375/37489389-d3b6131a-286d-11e8-9005-b0897d1d9b2e.png)

**Comment by jeff-phillips-18** [comment]
Issue is in the UI. Got a fix.

**Comment by jeff-phillips-18** [comment]
#1051 Fixes the list of hosts displayed in both the sources and scans views.

However, the number displayed in the Sources list is directly from the API and shows the total systems for the connection scan. There is no data returned that would allow us to determine which systems are for which source.

**Comment by kholdaway** [comment]
When calling `/api/v1/jobs/1/connection/` the source is part of the system object.

```{
    "count": 80,
    "next": "http://localhost:8000/api/v1/jobs/1/connection/?page=2",
    "previous": null,
    "results": [
        {
            "name": "01d0a4d93530-172.17.92.0-15-fakeprofile",
            "status": "success",
            "source": {
                "id": 3,
                "name": "S5Source",
                "source_type": "satellite"
            },
            "credential": {
                "id": 2,
                "name": "SCreds5"
            }
        }
    ]
}```

**Comment by jeff-phillips-18** [comment]
We use that to show the hosts in the expansion views, but to get that number in the list we would need to get the sources, then for each source, make an additional call in.

Added issue #1058 to address the remaining issue.

**Comment by bclarhk** [comment]
The issue still exists on the Scans page.
![image](https://user-images.githubusercontent.com/32572198/37921835-e1216a84-30f8-11e8-80ee-15b5f338bcb3.png)

**Comment by jeff-phillips-18** [comment]
The UI could fix the display of multiple instances of the same system by filtering out duplicates. However, the count in the list (12 above) would not coincide with the systems shown. This should be addressed on the API side if possible.

@chambridge @kholdaway

**Comment by chambridge** [comment]
So I appreciate the discussion above and the "feeling" that only 11 systems should show up, but the truth is that 12 system scans were performed. Because `10.10.181.98` exists in multiple sources it will be scanned twice creating two data points that will be used in the deduplication and merging portion that creates the summary report.

Would it be clearer to a user if we we also showed the associated source the system came from?
✅ **10.10.181.98**  _Net#1_
✅ **10.10.181.98**  _Net#2_

**Comment by jeff-phillips-18** [comment]
@bclarhk @serenamarie125 thoughts?  Sounds like a decent solution to me.

**Comment by bclarhk** [comment]
Is that the result that will be in the download? If so then 👍

**Comment by jeff-phillips-18** [comment]
Summary report only shows 10.10.181.98 once (Network #2), Detailed report shows it under both sources.
